### ISSUE TITLE
  false errors with php files

### ISSUE THREAD

**Issue by rino1** [DEVSENSE/phptools-docs#517]
````
$ cat test.php 
<?php
namespace private;

$ php -l test.php 
No syntax errors detected in test.php
````

But...

https://gcdnb.pbrd.co/images/B7uimQWB16lc.png?o=1

appears at the bottom

**Syntax error: unexpected token 'private'**

Why?

**Comment by jakubmisek** [comment]
Thank you for reporting the issue.

This is our bug; PHP <8.0 did not allow namespaces with reserved keywords, such as private.

We will fix this.

A quick workaround is to use a backslash, i.e. `namespace \private;`

**Comment by jakubmisek** [comment]
Fixed in `v1.45.15192`, thanks!

**Comment by rino1** [comment]
thanks
